src/app/shared/comment-editor/markdown-toolbar/markdown-toolbar.component.html

```diff
@@ -0,0 +1,64 @@
+<markdown-toolbar [attr.for]="forTextAreaId">
```

Comment:
Might be good to add comments to link to documentation for these html tags, either in html file itself with <!-- or in ts file.

Reply:
Okay, I've added a link to the repo of this library at the top of the HTML file.